Makefile

```
@@ -188,95 +149,23 @@ $(CR):
-export UPDATE_VALUES_YAML
-$(CHARTS_DIRECTORY)/lxcfs-on-kubernetes-$(RELEASE_VERSION).tgz: helm $(CHART_TEMPLATE_PATH)/clusterrole.yaml \
-	$(CHART_TEMPLATE_PATH)/certificate.yaml $(CHART_TEMPLATE_PATH)/issuer.yaml \
-	$(CHART_TEMPLATE_PATH)/mutatingwebhookconfiguration.yaml \
-	$(CHART_TEMPLATE_PATH)/deployment.yaml \
-	$(CHART_TEMPLATE_PATH)/daemonset.yaml \
-	$(CHART_TEMPLATE_PATH)/clusterrolebinding.yaml
-
-	sed -i.bak -E "$$UPDATE_CHART_YAML" ${CHART_PROJECT_PATH}/Chart.yaml
-	sed -i.bak -E "$$UPDATE_VALUES_YAML" ${CHART_PROJECT_PATH}/values.yaml
-	$(MAKE) $(CHART_PROJECT_PATH)/README.md
+$(CHARTS_DIRECTORY)/lxcfs-on-kubernetes-$(RELEASE_VERSION).tgz: helm
```

checkmake (0.3.2)

[warning] Target body for "$(CHARTS_DIRECTORY)/lxcfs-on-kubernetes-$(RELEASE_VERSION).tgz" exceeds allowed length of 5 lines (6).

(maxbodylength)